src/modules/poolModule.ts

```diff
@@ -36,7 +37,7 @@ import { getObjectPreviousTransactionDigest } from '../utils/objects'
 import { ClmmpoolsError, ConfigErrorCode, PartnerErrorCode, PoolErrorCode, PositionErrorCode, UtilsErrorCode } from '../errors/errors'
 
 type GetTickParams = {
-  start: number[]
+  start: number
```

Comment:
I wasn't 100% sure if I found all the right contract definitions for these calls, but I *think* this change makes sense